### PR TITLE
Migrations

### PR DESCRIPTION
**Pull request by MarcoAurelioAntonio**
## Blog app: Creating a Data Model
In this milestone I have:

- Installed `Ruby on Rails` framework.
- Used `Ruby Gems` as a software package system.
- Understood what `ORM` is.
- Used database migration files to maintain database `schema`.
- All the used `commands` are in every commit comment.

image

image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,201 @@
...
+
+  - [Answer_2]
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./MIT.md) licensed.
```
Comment (descholar-ceo) [inline README.md:199]: - [ ] This link is not working, could you fix it please?

File: README.md [hunk 2]
```diff
@@ -1,24 +1,201 @@
...
+### Key Features <a name="key-features"></a>
+
+- **[Creating a Data Model]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- in progress :smile: [Live Demo Link]()
```
Comment (descholar-ceo) [inline README.md:71]: - [ ] The live demo in this link is not working, you will need to fix the link

File: README.md [hunk 3]
```diff
@@ -1,24 +1,201 @@
...
+- Twitter: [@gustavoporcilio](https://twitter.com/gustavoporcilio)
+- LinkedIn: [Gustavo Porcilio](https://www.linkedin.com/in/gustavo-porcilio-4496a223a/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[]**
```
Comment (descholar-ceo) [inline README.md:152]: - [ ] You need to add three potential future features

File: README.md [hunk 4]
```diff
@@ -1,24 +1,201 @@
...
+
+- **[Question_1d]**
+
+  - [Answer_1]
+
+- **[Question_2]**
+
+  - [Answer_2]
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```
Comment (descholar-ceo) [inline README.md:193]: - [ ] Feel free to remove this section if it's empty

**Review by descholar-ceo — CHANGES_REQUESTED** [review]
Hi @MarcoAurelioAntonio 🤝 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
👍🏽 The PR is professional with a professional PR description ✔️ 
👍🏽 Linters installed with no errors ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
- [ ] Commit messages should always start with capital letter and make sure that they are imperative mood

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 2 above)
Comment (MarcoAurelioAntonio) [reply README.md:71]: That's because there is no live demo yet. I have no URL to fill the parentesis.

**Review by codecaiine — APPROVED** [review]
Hi @MarcoAurelioAntonio , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Great project implementation :heavy_check_mark: 
- Good use of gitflow :heavy_check_mark: 
- No rubocop errors :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@codecaiine) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**